packages/turndown/test/index.html

```diff
@@ -846,6 +846,40 @@ <h2>This is a header.</h2>
+    </div>
+  </div>
+  <pre class="expected">Hello World :
+
+$$
+\sqrt{3x}
+$$</pre>
+</div>
+
```

Comment:
We don't use these tests anymore, they are from the original source code that we've forked, so please remove these changes and move anything relevant to html_to_md directory

Reply:
Ohh ok, I will do that